files/en-us/web/api/baseaudiocontext/createbuffer/index.html

```diff
@@ -26,12 +26,12 @@
 
 <div class="note">
   <p><strong>Note</strong>: <code>createBuffer()</code> used to be able to take compressed
-    data and give back decoded samples, but this ability was removed from the spec,
-    because all the decoding was done on the main thread, therefore
+    data and give back decoded samples, but this ability is no longer in the specification,
```

Comment:
```suggestion
    data and give back decoded samples, but this ability was removed from the specification,
```

It's a minor point but I think "was removed from" is better than "is no longer in" (active versus passive voice). I agree that "spec" => "specification" is better.

Reply:
Sounds good!